mlrun/launcher/local.py

```diff
@@ -216,14 +216,15 @@ def _create_local_function_for_execution(
 
         meta = mlrun.model.BaseMetadata(function_name, project=project)
 
-        command, runtime = mlrun.run.load_func_code(
+        command, _runtime = mlrun.run.load_func_code(
```

Comment:
why tho you need `runtime` and `_runtime` scoped within same function? it is pretty error-prone

Reply:
fixed, added a comment to explain what I did